core/src/test/java/org/apache/calcite/sql/test/SqlAdvisorTest.java

```diff
@@ -1585,4 +1587,17 @@ private void testSimpleParserQuotedIdImpl() {
     assertSimplify(sql, simplified);
     assertComplete(sql, EXPR_KEYWORDS, Collections.singletonList("TABLE(a)"), DEPT_COLUMNS);
   }
+
+  private static SqlSimpleParser simpleParser =
+      new SqlSimpleParser("_suggest_", SqlParser.Config.DEFAULT);
```

Comment:
Make this a local variable, not a static field. Static fields use memory after the test has finished, and may not be reentrant.